plugins/MauticFocusBundle/Form/Type/ContentType.php

```diff
@@ -97,7 +100,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
 
         $builder->add(
             'font',
-            'choice',
+            ChoiceType::class,
```

Comment:
All choices must have the values and labels switched. See more details in https://docs.google.com/document/d/1lHNK8hHQtvyWlz9OtfAoLupDVb5-QunQjq-HAGlAsS4/edit